xml/System.Threading.Channels/Channel.xml

```diff
@@ -42,7 +42,7 @@
         <Parameter Name="capacity" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <typeparam name="T">Specifies the type of data in the channel.</typeparam>
+        <typeparam name="T">The data type in the channel.</typeparam>
```

Comment:
I think there's a difference between "data type" and "type of data".  The new version doesn't sound as correct to me.  I interpret "type of data" to mean there's some data in the channel and this is the type of it; I interpret "data type" to suggest that the channel actually contains types.

Reply:
Ok thanks. I'll revert it.

Reply:
I'll revert it to the original.
```suggestion
        <typeparam name="T">Specifies the type of data in the channel.</typeparam>
```